docs/source/contrib.rst

```diff
@@ -0,0 +1,44 @@
+Dask Contrib
+============
+
+In addition to the primary `Dask GitHub Organization <https://github.com/dask>`_ there is also a secondary
+`Dask Contrib GitHub Organization <https://github.com/dask-contrib/>`_ which contains community supported
+projects which integrate with or enhance the Dask experience.
+
+These projects are owned and maintained by folks outside of the core Dask maintainer group and therefore
+the quality and stability may be different to that of the main Dask packages. However these projects are
+expected to conform to the `Dask Code of Conduct <https://github.com/dask/governance/blob/main/code-of-conduct.md>`_.
+
+The contrib organisztion exists to improve discoverability of user submitted projects.
+
+Contrib package criteria
+------------------------
+
+If you are working on a project which you think could be a great addition to the Dask Contrib org then please
+feel free to raise an issue on the `Dask Community repo <https://github.com/dask/community>`_ requesting a transfer.
+
+For a project to be accepted into Dask Contrib it must meet the following criteria:
+
+- Packages must complement or enhance Dask in some way, maybe it contains plugins, adds integration with some other library or just generally improves the Dask experience for users.
+- Packages must have at least one active maintainer.
+- Packages must be installable both from PyPI with ``pip`` and Conda Forge with ``conda``.
+- Packages must have documentation.
+
+Graduation to the main org
+--------------------------
+
+In some cases it may be appropriate to graduate a project to the main Dask Organization, or perhaps merge the code/functionality
+into an existing Dask project.
+
+This will be assessed on a case-by-case basis by the Dask maintenance team.
+
+For some projects it may make sense to graduate their content into other repositories. The now defunct ``dask-lightgbm`` project was integrated into
```

Comment:
Just a wording suggestion - "defunct" has a negative connotation to me, but I don't feel strongly about this.

```suggestion
For some projects it may make sense to graduate their content into other repositories. The now archived ``dask-lightgbm`` project was integrated into
```